common.mk

```diff
@@ -196,11 +196,13 @@ CROSS_COMPILE_NS_USER   ?= "$(CCACHE)$(AARCH$(COMPILE_NS_USER)_CROSS_COMPILE)"
 CROSS_COMPILE_NS_KERNEL ?= "$(CCACHE)$(AARCH$(COMPILE_NS_KERNEL)_CROSS_COMPILE)"
 CROSS_COMPILE_S_USER    ?= "$(CCACHE)$(AARCH$(COMPILE_S_USER)_CROSS_COMPILE)"
 CROSS_COMPILE_S_KERNEL  ?= "$(CCACHE)$(AARCH$(COMPILE_S_KERNEL)_CROSS_COMPILE)"
+CROSS_COMPILE_RUST	?= "$(AARCH$(COMPILE_S_KERNEL)_CROSS_COMPILE)"
```

Comment:
This is user space only so `COMPILE_S_USER` would be more appropriate.

Reply:
Makes sense, I will use that instead.